pegasus/sites.v3/code.org/themes/responsive_full_width.haml

```diff
@@ -1,6 +1,6 @@
 !!! 5
 
-- if @header['set_dir']
+- if ! @header['set_dir']
```

Comment:
Do we need the `if` statement anymore or can we just have `- dir = language_dir_class`? I'm a bit confused on how adding the `!` adds this functionality

Reply:
I think it might have to do with the ability to set this functionality on individual pages with the `set_dir: true` seen in the About page file in this PR. 

If there comes a day when this can be added sitewide, all this logic can go away 😅

Reply:
Will this break any pages with `set_dir` as `false` or `nil` since that will then pass this condition and set `dir` to `language_dir_class`? Or is that the intention? Sorry just want to understand!

Reply:
Oh great catch! It will break on pages where `set_dir: true` (it still works when set_dir: false or nil), so I removed that setting from two recently updated pages [in this commit](https://github.com/code-dot-org/code-dot-org/pull/57421/commits/7c3a3205b67d98ba3697ab5e084e9b29dbaaa528). None of the other pages where this is applied uses that `set_dir` setting so they should be good.